src/emscripten-optimizer/parser.cpp

```diff
@@ -41,6 +41,7 @@ IString TOPLEVEL("toplevel"),
         CONTINUE("continue"),
         SWITCH("switch"),
         STRING("string"),
+        TRY("try"),
```

Comment:
I use try...catch in the translation of assert_trap to JS. The translation of assert_trap tests is behind a --pedantic flag because asm.js does not trap in places where wasm does.